manifests/init.pp

```diff
@@ -26,7 +26,7 @@
 # }
 class nginx (
   ### START Nginx Configuration ###
-  $client_body_temp_path                                     = $nginx::params::client_body_temp_path,
+  Variant[Stdlib::Absolutepath, Boolean] $client_body_temp_path = $nginx::params::client_body_temp_path,
```

Comment:
I wasn't sure about a boolean false here or an empty string. Thoughts?

Reply:
I think `Boolean` is better than empty string and it's probably not worth making it a breaking change by defaulting to `undef` and making it `Optional`.

Reply:
It does mean that true is allowed but not really handled - OTOH, it wasn't either in the past